### PR TITLE
ITSD-11413: cell text modification history

### PR DESCRIPTION
**Pull request by StefanRoggenstein**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -94,9 +94,15 @@ class Spreadsheet {
     return this.dataSet.map((it) => it.getData());
   }
 
-  cellText(ri, ci, text, force = false, sheetIndex = 0) {
+  cellText(ri, ci, text, force = false, saveHistory = false, sheetIndex = 0) {
```
Comment (aleschneck) [inline src/index.js]: why is this per default `false`, should it not be `true`?

Comment (StefanRoggenstein) [reply src/index.js]: In our VDI app, we frequently utilize this function to write error and warning texts in cells that are not required to be saved in the history. Approximately 95% of the time, we do not need these texts to be saved, while only about 5% of the time, we actually require them to be preserved. Therefore, it is more efficient and storage-friendly to have this setting as false by default.

Comment (aleschneck) [reply src/index.js]: it nevertheless begs the question as to what is the expected behaviour and in my opinion it should be the other way around regardless of how VDI uses it

Comment (StefanRoggenstein) [reply src/index.js]: Okay, I agree. It's changed now.

**Review by aleschneck — APPROVED** [review]

**Pull request merged**